### PR TITLE
Add a maxHeight to externalImage component

### PR DESCRIPTION
**Pull request by BrodyHughes**
The bottom of the NFT images has a slight gap. The external Image component had a slightly larger height than expected. This should fix it. 
| Before | After |
|--------|-------|
| Screenshot 2023-10-04 at 3 03 12 PM | Screenshot 2023-10-04 at 3 34 44 PM |
| Screenshot 2023-10-04 at 3 35 00 PM | Screenshot 2023-10-04 at 3 40 40 PM |

### CODE REVIEW

File: src/entries/popup/components/CoinIcon/CoinIcon.tsx [hunk 1]
```diff
@@ -282,12 +282,13 @@ export const NFTIcon = ({
         width={size}
         borderRadius={nftRadiusBySize[size]}
         onError={() => setBadSrc(true)}
+        maxHeight={size}
       />
-      <Box position="absolute" bottom="0" style={{ zIndex: 2, left: '-6px' }}>
-        {badge && chainId !== ChainId.mainnet && (
+      {badge && chainId !== ChainId.mainnet && (
```
Comment (BrodyHughes) [inline src/entries/popup/components/CoinIcon/CoinIcon.tsx:287]: also moved this to not render this empty box for the NFT components

Comment (greg-schrammel) [reply src/entries/popup/components/CoinIcon/CoinIcon.tsx:287]: nicee

File: src/entries/popup/components/ExternalImage/ExternalImage.tsx [hunk 2]
```diff
@@ -60,7 +61,13 @@ const ExternalImage = (
 
   if (!signedUrl) return null;
   return (
-    <Box style={{ overflow: 'clip' }} borderRadius={props.borderRadius}>
+    <Box
+      style={{
+        overflow: 'clip',
+        maxHeight: props.maxHeight ? props.maxHeight : '',
```
Comment (greg-schrammel) [inline src/entries/popup/components/ExternalImage/ExternalImage.tsx:67]: I think we can safely set `maxHeight` to `height`, so we don't need the extra `maxHeight` prop

Comment (BrodyHughes) [reply src/entries/popup/components/ExternalImage/ExternalImage.tsx:67]: ok cool. wasn't sure if this was a good idea or not / had other implications elsewhere that i hadn't considered.

Comment (BrodyHughes) [reply src/entries/popup/components/ExternalImage/ExternalImage.tsx]: fixed.

**Review by brunobar79 — APPROVED** [review]
💯

**Review by greg-schrammel — APPROVED** [review]

**Pull request merged**